cypress/integration/unit_tests/checkbox_test.js

```diff
@@ -315,6 +328,114 @@ describe('Unit test for toggleLayerOn', () => {
           expect(props.id).to.eql(scoreLayerName);
         });
   });
+
+  // TODO(janakr): For all the composite tile tests below, we perform 4 fetches
+  //  for each tile url, but only 2 unique ones. This doesn't happen in
+  //  production, so it's not a performance issue, seemingly just something
+  //  weird about the requests Google Maps makes in tests.
+  it('tests composite tiles', () => {
+    // Stub out HTTP fetch, so we can return a promise that waits a while.
+    const fetchStub = cy.stub(window, 'fetch');
+    // Only release the promise when we've made all our assertions.
+    let releaseResponse = null;
+    const waitPromise = new Promise((resolve) => releaseResponse = resolve);
+    fetchStub.callsFake((url) => {
+      if (url.startsWith('tile-url1')) {
+        return okHttpResponse;
+      }
+      return waitPromise;
+    });
+    let map = null;
+    let overlay;
+    let addLayerPromise = null;
+    createGoogleMap().then((returnedMap) => map = returnedMap);
+    expectNoBlobImages().then(() => {
```

Comment:
why are these not chained?

Reply:
Cypress promises don't need to be chained, because Cypress handles them in order. Same reason you don't need to do cy.get('first').then(() => cy.get('second'));